tests/Jobs/UpdateWikiDailyMetricJobTest.php

```diff
@@ -39,12 +39,20 @@ public function testRunJobForAllWikisIncludingDeletedWikis()
 
         $this->assertDatabaseHas('wiki_daily_metrics', [
             'wiki_id' => $activeWiki->id,
-            'date' => Carbon::today()->toDateString()
+            'date' => Carbon::today()->toDateString(),
```

Comment:
We talked about how we could "properly" test this and concluded that this is one of the main weaknesses of this approach. We're so tightly bound to the implementation / layout of mediawiki's private dbs that this isn't really possible to seed with good test data. Hence why this just lives as a null